### PR TITLE
Correct unclear description in feature-online-ddl

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
Correct unclear description in feature-online-ddl

### Which DM version(s) do your changes apply to? (Required)

Since December 23, 2021, the TiDB DM documentation in the master branch has been merged to TiDB documentation by [#8042](https://github.com/pingcap/docs-cn/pull/8042) and [#7317](https://github.com/pingcap/docs/pull/7317). To create a PR for TiDB DM versions later than v5.3, either go to TiDB [English documentation repository](https://github.com/pingcap/docs) or [Chinese documentation repository](https://github.com/pingcap/docs-cn).

- [x] v5.3 (TiDB DM 5.3 versions)
- [ ] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-dm/issues/886

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/feature-online-ddl.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This document introduces the `online-ddl` feature of DM when DM is used to migra
 
 ## Overview
 
-DDL statements are always used in the database applications. MySQL 5.6 and later versions support `online-ddl` feature, but there are limitations for usage. For example, to acquire the MDL lock, some DDLs still need to be copied. In production scenarios, the table lock during DDL execution can block the reads or writes to and from the database to a certain extent.
+DDL statements are always used in the database applications. MySQL 5.6 and later versions support `online-ddl` feature, but there are limitations for usage. For example, to acquire the MDL lock, some DDLs still need tables to be copied. In production scenarios, the table lock during DDL execution can block the reads or writes to and from the database to a certain extent.
```
Comment (shichun-0415) [inline en/feature-online-ddl.md:12]:
```suggestion
DDL statements are always used in database applications. MySQL 5.6 and later versions support the `online-ddl` feature, but there are limitations. For example, DDL execution sometimes acquires an MDL lock, resulting in table lock that can block reads or writes to and from a database to a certain extent in production scenarios. In other cases, some DDLs copy the whole table and this affects overall database performance.
```

**Review by lichunzhu — APPROVED** [review]
LGTM. Great job!

File: en/feature-online-ddl.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This document introduces the `online-ddl` feature of DM when DM is used to migra
 
 ## Overview
 
-DDL statements are always used in the database applications. MySQL 5.6 and later versions support `online-ddl` feature, but there are limitations for usage. For example, to acquire the MDL lock, some DDLs still need to be copied. In production scenarios, the table lock during DDL execution can block the reads or writes to and from the database to a certain extent.
+DDL statements are always used in database applications. MySQL 5.6 and later versions support the `online-ddl` feature, but there are limitations. For example, DDL execution sometimes acquires an MDL lock, resulting in table lock that can block reads or writes to and from a database to a certain extent in production scenarios. In other cases, some DDLs copy the whole table and this affects overall database performance.
```
Comment (hfxsd) [inline en/feature-online-ddl.md:12]:
```suggestion
DDL statements are always used in database applications. MySQL 5.6 and later versions support the `online-ddl` feature, but there are limitations. For example, DDL execution sometimes acquires an MDL lock, resulting in a table lock that can block reads or writes to and from a database to a certain extent in production scenarios. In other cases, some DDLs copy the whole table and this affects overall database performance.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: zh/feature-online-ddl.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ title: 迁移使用 GH-ost/PT-osc 的源数据库
 
 ## 概述
 
-DDL 是数据库应用中必然会使用的一类 SQL。MySQL 虽然在 5.6 的版本以后支持了 online-ddl 功能，但是也有或多或少的限制。比如 MDL 锁的获取，某些 DDL 还是需要以 Copy 的方式来进行，在生产业务使用中，DDL 执行过程中的锁表会一定程度上阻塞数据库的读取或者写入。
+DDL 是数据库应用中必然会使用的一类 SQL。MySQL 虽然在 5.6 的版本以后支持了 online-ddl 功能，但是也有或多或少的限制。比如 MDL 锁的获取，某些 DDL 还是需要以 Copy 表的方式来进行，在生产业务使用中，DDL 执行过程中的锁表会一定程度上阻塞数据库的读取或者写入。
```
Comment (shichun-0415) [inline zh/feature-online-ddl.md:11]:
```suggestion
DDL 是数据库应用中必然会使用的一类 SQL。MySQL 虽然在 5.6 的版本以后支持了 online-ddl 功能，但是也有或多或少的限制。比如某些时候执行 DDL 会获取 MDL 锁，造成锁表，生产环境中，锁表会一定程度阻塞数据库的读取或写入。另外，某些 DDL 需要复制整个表，从而影响数据库整体性能。
```

**Pull request merged**